index.html

```diff
@@ -67,16 +77,39 @@ <h2>Software Developer</h2>
                 <li class="skill-list"><i >Codekit</i></li>
                 <li class="skill-list"><i >Codeopen</i></li>
             </ul>
-            
-            <div class="bootom-shine-1">
-                <img src="./img/circle_2.png" alt="Shine circle">
-            </div>
+
         </div>
 
         <div class="break">
             <img src="./img/hor_line.png" alt="Horizontal Line">
         </div>
 
+        <div class="social-links">
+            <div class="next-page">
+                <ul class="down-key">
+                    <li><img src="./img/down_icon.png" alt="Down Icon"></li>
+                    <li><span>works</span></li>
+                </ul>
```

Comment:
- [ ] **Required:** Please switch the icon and "Works ⬇️" to be similar to the Figma Template, then add more padding between the line break image and the social links here.

Your Design | Figma Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/150638542-3588b6dc-60d8-4529-8f96-f8db18af73ce.png) | ![image](https://user-images.githubusercontent.com/27709832/150638566-d81c09e1-4826-4b56-a87e-78a5b9714ff3.png)

- [ ] **Required:** Ensure you center this social icons to look very similar to the Figma Template